app/main/views/frameworks.py

```diff
@@ -85,17 +85,46 @@ def framework_dashboard(framework_slug):
     if countersigned_framework_agreement_exists_in_bucket(framework_slug, current_app.config['DM_AGREEMENTS_BUCKET']):
         countersigned_agreement_file = COUNTERSIGNED_AGREEMENT_FILENAME
 
+    application_made = supplier_is_on_framework or (len(complete_drafts) > 0 and declaration_status == 'complete')
+    lots_with_completed_drafts = [lot for lot in framework['lots'] if count_drafts_by_lot(complete_drafts, lot['slug'])]
+
+    last_modified = {
+        'supplier_pack': get_last_modified_from_first_matching_file(
+            key_list, framework_slug, "communications/{}".format(supplier_pack_filename)
+        ),
+        'supplier_updates': get_last_modified_from_first_matching_file(
+            key_list, framework_slug, "communications/updates/"
+        )
+    }
+
+    # if framework_agreement_version and application_made, show new framework dashboard template
+    if supplier_is_on_framework and framework['frameworkAgreementVersion'] and supplier_framework_info['agreementReturned']:  # noqa
+        agreements_bucket = s3.S3(current_app.config['DM_AGREEMENTS_BUCKET'])
+        signature_page = get_most_recently_uploaded_agreement_file_or_none(agreements_bucket, framework_slug)
+
+        # The default view (download_agreement_file) uses the DM_SUBMISSIONS_BUCKET, which doesn't work
+        signature_page_url = get_signed_url(agreements_bucket, signature_page['path'], 'https://{}.s3-eu-west-1.amazonaws.com'.format(current_app.config['DM_AGREEMENTS_BUCKET']))  # noqa
```

Comment:
We shouldn't hardcode the S3 URL here. It should be `current_app.config['DM_ASSETS_URL']` to go through nginx in live environments.